e2e-tests/cypress/tests/integration/channels/collapsed_reply_threads/channel_notifications_spec.ts

```diff
@@ -251,6 +251,7 @@ describe('CRT Desktop notifications', () => {
 
             // # Delete the replies
             cy.wrap(['@reply', '@replyMention']).each((reply) => {
+                // @ts-expect-error: reply should be of type PostMessageResp
```

Comment:
Instead of hiding the error caused by the odd typing, could we rewrite this to use a regular for loop instead of wrapping the array using Cypress? 

@harshilsharma63 CCing you since you're the one who added these tests originally

Reply:
```ts
const replies = ['@reply', '@replyMention'];
for(const reply of replies){
// more code here
}
```